docs/source/io.rst

```diff
@@ -17,6 +17,45 @@ Video
 .. autofunction:: write_video
 
 
+Low-level video API
+-------------------
+
+In addition to the :mod:`read_video` function, we provide a high-performance 
+low-level API for more fine-grained control compared to the old API.
+We expose it to users using TORCHBIND C++ registration, and allow users
+to access it via :mod:`torch.classes` import or via :mod:`torchvision.io`. For example
+
+.. autoclass:: Video
+    :members: next, get_metadata, set_current_stream, seek
+
+
+Example of usage:
+
+.. code:: python
+
+    import torch
+    import torchvision
+    video_path = "path to a test video"
+    # Constructor allocates memory and a threaded decoder
+    # instance per video. At the momet it takes two arguments:
+    # path to the video file, and a wanted stream.
+    
+    reader = torch.classes.torchvision.Video(video_path, "video")
+    # equivalently, on could call
```

Comment:
Let's remove this, we don't want to advertise this via `torch.classes`, the same way we don't advertise `nms` via `torch.ops.torchvision.nms`